protos/consensus.proto

```diff
@@ -188,8 +184,17 @@ message ConsensusNotifyAck {}
 
 // Send a consensus message to a specific, connected peer
 message ConsensusSendToRequest {
-  ConsensusPeerMessage message = 1;
-  bytes peer_id = 2;
+  // Payload to send to peer
+  //
+  // NOTE: This payload will be wrapped up in a ConsensusPeerMessage struct,
```

Comment:
Probably no need for 'NOTE:' -- that's what a comment is...

Reply:
But this is a note worthy comment.